<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow1</class>
 <widget class="QMainWindow" name="MainWindow1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Схема расстановки взвода</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
      <item>
       <widget class="QPushButton" name="transform_button">
        <property name="text">
         <string>Transform</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Download file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="download"/>
      </item>
      <item>
       <widget class="QPushButton" name="download_button">
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Load file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="load"/>
      </item>
      <item>
       <widget class="QPushButton" name="load_button">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>457</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
